ddl/ddl_api.go

```diff
@@ -1142,7 +1142,7 @@ func columnDefToCol(ctx sessionctx.Context, offset int, colDef *ast.ColumnDef, o
 				}
 				col.GeneratedExprString = sb.String()
 				col.GeneratedStored = v.Stored
-				_, dependColNames := findDependedColumnNames(colDef)
+				_, dependColNames, _ := findDependedColumnNames("", "", colDef)
```

Comment:
Why not checking the possible error?

Reply:
Updated. It was checked before, but I think rechecking `error` here makes it easier to read.